integration_tests/pd_api_test.go

```diff
@@ -49,7 +50,7 @@ type apiTestSuite struct {
 func (s *apiTestSuite) SetupTest() {
 	require := s.Require()
 	addrs := strings.Split(*pdAddrs, ",")
-	pdClient, err := pd.NewClient(addrs, pd.SecurityOption{})
+	pdClient, err := pd.NewClient(caller.Component("client-go-integration-test"), addrs, pd.SecurityOption{})
```

Comment:
@okJiang, do we have a standard for the definition of the component?

Reply:
If it is in the test, we can use caller.TestComponent. Otherwise, we can put it empty now, I will update them in https://github.com/tikv/client-go/pull/1516

Reply:
BTW, how about adding a `DefaultComponent = ""` in pd repo?